webpack.config.js

```diff
@@ -1,8 +1,10 @@
 const path = require('path');
 const HmtlWebpackPlugin = require('html-webpack-plugin');
+// const { check } = require('lockfile');
```

Comment:
- [ ]  Please kindly remove commented code from you code base as this makes you work dirty and less readable.

Reply:
Thank you @Mihndim2020 for your suggestions! I removed these comments from my code.